packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -13,41 +22,115 @@ export interface ExchangeConfirmationCardProps {
 type Props = ExchangeConfirmationCardProps
 
 export default function ExchangeConfirmationCard(props: Props) {
+  const { t } = useTranslation(Namespaces.exchangeFlow9)
   const { makerAmount, takerAmount } = props
+  const isSellGoldTx = makerAmount.currencyCode === CURRENCIES[CURRENCY_ENUM.GOLD].code
+  const [gold, dollars] = isSellGoldTx
+    ? [makerAmount.value, takerAmount.value]
+    : [takerAmount.value, makerAmount.value]
+
+  // TODO: show real fees
+  const tobinTax = 0
+  const fee = 0
+
+  const localAmount = (isSellGoldTx ? makerAmount : takerAmount).localAmount!
+  // TODO: find a way on how to show local exchangeRate without this hack
+  const exchangeRateAmount = {
+    value: localAmount.exchangeRate,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+    localAmount: {
+      value: localAmount.exchangeRate,
+      exchangeRate: localAmount.exchangeRate,
+      currencyCode: localAmount.currencyCode,
+    },
+  }
+
+  const goldAmount = {
+    value: gold,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.GOLD].code,
+  }
+
+  const subtotalAmount = {
+    value: dollars,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+  }
+
+  const exchangeFeeAmount = {
+    value: tobinTax,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+  }
+  const securityFeeAmount = {
+    value: fee,
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+  }
+
+  const totalAmount = {
+    value: new BigNumber(dollars).plus(tobinTax).plus(fee),
+    currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+  }
 
   return (
-    <View style={styles.container}>
-      <View style={styles.exchange}>
-        <CurrencyDisplay type={DisplayType.Big} amount={makerAmount} size={36} useColors={true} />
-        <View style={styles.arrow}>
-          <RoundedArrow />
-        </View>
-        <CurrencyDisplay type={DisplayType.Big} amount={takerAmount} size={36} useColors={true} />
+    <SafeAreaView style={styles.container}>
+      <View style={styles.paddedContainer}>
+        <ScrollView>
```

Comment:
For fullscreen views, it's nicer to put `<SafeAreaView>` inside the `<ScrollView>` so when scrolling up and down the content is not clipped before the bottom/top of the screen.

Reply:
Would content overlap with the bar in the bottom on iPhone X in this case?

Reply:
Yes but that's what is preferred visually, the difference is that there's some padding at the bottom so you can still fully view all the content from the scrollview without overlap.